README.md

```diff
@@ -130,5 +130,15 @@ Sample query to fetch events for given patient AND tag:
 /events?select=event&event.tags=cs.{mme-calc}&event-%3E%3Epatient=eq.marcus%20aurelius
 ```
 
+Fetch events based on datetime:
+```http request
+/events?select=event->>asctime,event->>message&event->>asctime=gte.2023-04-01&limit=25
+```
+
+Fetch events based on datetime, and include a number of isacc-specific fields:
```

Comment:
Log server spans many products - suggest `resource-specific` in place of `isacc-specific`